content/pages/talks.html

```diff
@@ -1,16 +1,44 @@
 ---
-$title@: Talks
+$title@: Presentations
 $hidden: true
+$path: /presentations/
 ---
 
 <div class="header about-header">
-    <h2>{{doc.title}}</h2>
+    <h2>Presentations</h2>
 </div>
+{% set highlighted = 'talks' %}
```

Comment:
Unfortunately, it doesn't quite work that way.  Take a look at the about_nav.  In the short term, let's just add some more custom formatting for presentations.